addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/NewPlanDialog.java

```diff
@@ -73,6 +74,14 @@
         PassiveScanWaitJob.JOB_NAME,
         REPORT_JOB_NAME
     };
+    private static final String[] IMPORT_PROFILE = {
+        AddOnJob.JOB_NAME,
+        PassiveScanConfigJob.JOB_NAME,
+        "import",
+        PassiveScanWaitJob.JOB_NAME,
```

Comment:
I think its worth adding the Spider job here (after the "import" one).
We dont do that for soap/openapi profiles because they are API definitions - in this case we are importing list of urls so I think spidering makes much more sence.
People will be able to change the joibs of course, but in this case I think including the spider by default would be worth it.